added new modal for contact us iframe with discover.entando links

ianasqazi opened the pull request.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/entando-samples/entando-hub?pullRequest=149) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/entando-samples/entando-hub?pullRequest=149) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/entando-samples/entando-hub?pullRequest=149) it.</sub>

nshaw commented:
@ianasqazi A couple things. 1) Please sign the CLA if you haven't already. 2) Please start PRs and commit messages with the associated ticket number, e.g. EHUB-240 in this case.

nshaw commented on application/ui/widgets/eh-widgets-dir/eh-widgets/src/page/bundle-group/BundleGroupPage.js:
```diff
@@ -121,14 +132,28 @@ const BundleGroupPage = () => {
-                        <hr/>
-                      </>
+                    {(pageModel.bundleGroup.displayContactUrl) && (pageModel.bundleGroup.contactUrl)
+                        && checkContactUsUrl(pageModel.bundleGroup.contactUrl) == false
```
Getting eslint warnings here on startup, please switch to ===

ianasqazi replied:
Updated to ===

nshaw commented on application/ui/widgets/eh-widgets-dir/eh-widgets/src/page/bundle-group/modal-contact-us-information/ModalContactUsInformation.js:
```diff
@@ -0,0 +1,66 @@
+  return (
+      <div className="Modal-contact-us-code">
+
+          <iframe src={bundleGroup.contactUrl} style={{width:'100%',height:'400px'}}/>
```
Please add a title to satisfy eslint so we can keep the build clean. 
  _Line 11:11:  <iframe> elements must have a unique title property  jsx-a11y/iframe-has-title_

ianasqazi replied:
Added Title attribute for iframe tags

nshaw commented on application/ui/widgets/eh-widgets-dir/eh-widgets/src/page/bundle-group/BundleGroupPage.js:
```diff
@@ -88,6 +89,16 @@ const BundleGroupPage = () => {
   }, [bundleGroupVersionId])
 
+  // checks the contact-us url for discover
+  const checkContactUsUrl = (string) => {
```
Minor, this can be simplified.
const useContactUsModal = (url) => {
   return url && url.includes("discover.entando.com")
}

ianasqazi replied:
Updated to the simplified version as above. 
Thank you

ianasqazi commented:
- I have updated the commit messages to include the ticket number
- Signed the CLA agreement (waiting for the status to change)
- Updates made for eslint fixes and simplified check modal function

ianasqazi commented:
@nshaw Updated the PR 
Change log : Merged the different scss files into single scss file for all information modal properties.

nshaw commented on application/ui/widgets/eh-widgets-dir/eh-widgets/src/page/bundle-group/BundleGroupPage.js:
```diff
@@ -121,14 +127,28 @@ const BundleGroupPage = () => {
-                        </div>
-                        <hr/>
-                      </>
+                    {(pageModel.bundleGroup.displayContactUrl) && (pageModel.bundleGroup.contactUrl)
```
@anasliferay This logic isn't quite right. I'm getting a Contact Us modal on ALL URLs now when displayContactUrl and contactUrl are not set because the else always executes.

ianasqazi replied:
@nshaw updated the condition to not display the CONTACT US button at all when there is no contact URL based on length.
